js/ui/workspace.js

```diff
@@ -1022,6 +1022,13 @@ const Workspace = new Lang.Class({
     positionWindows: function(flags) {
         this._positionWindowsFlags |= flags;
 
+        /* This might not be the best place for this check. It prevents the
+         * shell from crashing in _computeAllWindowSlots when launching the
+         * tutorial if the actor has not been added to the stage. See T3727.
+         */
+        if (!this.actor.mapped)
```

Comment:
Yeah, this does not seem the best place for the check...
What kind of backtrace did you get? Does the crash happen every time?